protobufs/protocol/protocol.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
this is an envelope for all p2p protobufs so that they can be identified and decoded.

Reply:
needs to be moved to centrifuge-protobufs , its here for the sake of hackability.